check_maintenance_incidents.py

```diff
@@ -29,6 +30,17 @@ def add_devel_project_review(self, req, package):
         if action_is_patchinfo(a):
             a = req.actions[1]
         project = a.tgt_releaseproject if a.type == 'maintenance_incident' else req.actions[0].tgt_project
+
+        if project.startswith('openSUSE:'):
```

Comment:
In the request that was quoted in the mailthread this project should be `openSUSE:Maintenance` and that probably is always the case. why are we testing with startswith(openSUSE:) which is very broad rather than exactly the project name?

Reply:
Well, this is for maintenance check. The `/search/owner` backend here fails for every package that is in factory and doesn't have devel project set in OBS. And AFAIK, we do not have devel project configured for other projects than openSUSE:Factory. So, everything that is sent as maintenance update to any openSUSE: project that uses this bot will no longer work if openSUSE:Factory devel projects are not set.

If you would like to limit this to `openSUSE:Maintenance` only, that's fine and I can change this accordingly, but `/search/owner` will not really function against other projects (we have the attribute for it set in only one owning project, openSUSE:).